integration/render_test.go

```diff
@@ -350,60 +326,42 @@ spec:
       labels:
         app: skaffold-helm
         release: skaffold-helm
+        skaffold.dev/run-id: phony-run-id
     spec:
       containers:
-        - name: skaffold-helm
-          image: gcr.io/k8s-skaffold/skaffold-helm:sha256-nonsenslettersandnumbers
-          imagePullPolicy: Always
-          ports:
-            - containerPort: 80
-          resources:
-            {}
+      - image: gcr.io/k8s-skaffold/skaffold-helm:sha256-nonsenselettersandnumbers
+        imagePullPolicy: always
+        name: skaffold-helm
+        ports:
+        - containerPort: 80
+        resources: {}
 ---
-# Source: skaffold-helm/templates/ingress.yaml
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: skaffold-helm-skaffold-helm
+  annotations: null
```

Comment:
nit: why annotations: null vs what it was?

Reply:
I'm assuming this has to do with a change in helm 3.x and iterating of lists in templates? But it seems that it provides null as the value when there is an empty list of values